### PR TITLE
[Docs] K8s Configuration Doc edits

### PR DESCRIPTION
**Pull request by bcrant**
Just some tiny tweaks and fixes to the docs.

### CODE REVIEW

**Comment by liranbg** [comment]
> Just some tiny tweaks and fixes to the docs.

Thanks for your contribution @bcrant

File: docs/runtimes/mlrun_jobs.ipynb [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "- [Define a new function and its dependencies](#define-a-new-function-and-its-dependencies)\n",
     "- [Convert the code to a serverless job](#convert-the-Code-to-a-serverless-job)\n",
     "- [Deploy (build) the function container](#deploy-build-the-function-container)\n",
-    "- [Run the function on the cCluster](#run-the-function-on-the-cluster)\n",
+    "- [Run the function on the Cluster](#run-the-function-on-the-cluster)\n",
```
Comment (jillnogold) [inline docs/runtimes/mlrun_jobs.ipynb:15]: Run the function on the cluster
Lower case!

File: docs/runtimes/configuring-job-resources.md [hunk 2]
```diff
@@ -113,12 +113,12 @@ Node selector is supported for all cloud platforms. It is relevant for MLRun and
...
 When running ML functions you might want to control whether to run on spot nodes or on-demand nodes. Preemption mode controls whether pods can be scheduled on preemptible (spot) nodes. Preemption mode is supported for all functions. 
 
-Preemption mode uses Kubernets Taints and Toleration to enforce the mode selected. Read more in [Kubernetes Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration). 
+Preemption mode uses Kubernetes Taints and Toleration to enforce the mode selected. Read more in [Kubernetes Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration). 
 
 ### Why preemption mode
 
 On-demand instances provide full control over the instance lifecycle. You decide when to launch, stop, hibernate, start, 
-reboot, or terminate it. With Spot instances you request capacity from specific available zones, though it is  
+reboot, or terminate it. With Spot instances, you request capacity from specific availability zones, though it is
```
Comment (jillnogold) [inline docs/runtimes/configuring-job-resources.md:121]: *from specific available zones*

File: docs/runtimes/configuring-job-resources.md [hunk 3]
```diff
@@ -204,7 +205,7 @@ Pods (services, or jobs created by those services) can have priorities, which in
 scheduling: a lower priority pod can be evicted to allow scheduling of a higher priority pod. Pod priority is relevant for all pods created 
 by the service. For MLRun, it applies to the jobs created by MLRun. For Nuclio it applies to the pods of the Nuclio-created functions.
 
-Eviction uses these values to determine what to evict with conjunction to the pods priority [Pod Priority and Preemption](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption).
+Eviction uses these values in conjuction with pod priority to determine what to evict [Pod Priority and Preemption](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption).
```
Comment (jillnogold) [inline docs/runtimes/configuring-job-resources.md:208]: Eviction uses these values in conjunction with pod priority to determine what to evict

File: docs/runtimes/mlrun_jobs.ipynb (hunk 1 above)
Comment (bcrant) [reply docs/runtimes/mlrun_jobs.ipynb]: Good catch, fixed

File: docs/runtimes/configuring-job-resources.md (hunk 2 above)
Comment (bcrant) [reply docs/runtimes/configuring-job-resources.md:121]: Hi @jillnogold, I am not sure if your comment is a correction or just stating what it said before. Would you like me to revert [availability zones](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/) back to `available` zones?

File: docs/runtimes/configuring-job-resources.md (hunk 3 above)
Comment (bcrant) [reply docs/runtimes/configuring-job-resources.md:208]: Hi @jillnogold, not sure if your comment is meant to be actionable or just for your own notes, but that is was the text was changed to (green)

File: docs/runtimes/configuring-job-resources.md (hunk 2 above)
Comment (jillnogold) [reply docs/runtimes/configuring-job-resources.md:121]: Hi @bcrant - Thanks for your input! And please ignore that comment.

File: docs/runtimes/configuring-job-resources.md (hunk 3 above)
Comment (jillnogold) [reply docs/runtimes/configuring-job-resources.md:208]: Hi @bcrant - looks like twice I hit Comment instead of cancel.... Sorry for the confusion

**Comment by bcrant** [comment]
> Hi @bcrant - Can you please resolve the conflicts?

done!

**Pull request merged**